net/zerotier/patches/0005-do-not-use-zt-SSO-and-OIDC.patch

```diff
@@ -0,0 +1,35 @@
+From c368d51cb1d221cf3ffb2d33d03dcc69591339c0 Mon Sep 17 00:00:00 2001
+From: Moritz Warning <moritzwarning@web.de>
```

Comment:
Shouldn't be here author @oskarirauta ?

Reply:
sure

Reply:
Thanks for credits although not important on my behalf, glad that I could help.
I will answer to question 2 with quotation from my previous post..

> _with these changes changes it builds successfully a universal version - meaning a version that it builds for some architectures that do not support ZT_SSO, so with patch it builds such version for all archs - I do not know what ZT_SSO is and how important/useful it is, but it seems to be quite clear that in future it most likely will be the only way forward with zerotier and therefore we should prepare for it.. So work on rust/cargo..._

So could be merged upstream but won't.

Reply:
Okay, sorry, I did not go through all the discussion here, but this should be definitely included in the commit message of the patch.